index.test.js

```diff
@@ -98,15 +98,20 @@ test.only('bodymovin.json => mp4', async (t) => {
 
   await renderLottie({
     path: bodymovin,
-    quiet: true,
-    output
+    quiet: false,
```

Comment:
let's keep the test output quiet :)

Reply:
Ok, my bad, I forgot to put it back to quiet.